index.bs

```diff
@@ -154,7 +154,7 @@ Reference Frame {#reference-frame}
 ----------------
 
 The reference frame for the sensor classes defined in this specification,
```

Comment:
Sorry for the late review. Would this be a simpler way to specify the same:

```
The [=local coordinate system=] represents the <dfn>reference frame</dfn> for the [=readings=].
It can be either the [=device coordinate system=] or the [=screen coordinate system=].
```

(dfn allows cross-linking "reference frame" from https://w3c.github.io/sensors/#concepts-sensors)

Reply:
Updated the comment above to not make "reference frame" a dfn since we use the standard definition of the term https://en.wikipedia.org/wiki/Frame_of_reference